mesonbuild/mtest.py

```diff
@@ -1124,15 +1124,13 @@ async def read_decode(reader: asyncio.StreamReader, console_mode: ConsoleUser) -
 # Extract lines out of the StreamReader.  Print them
 # along the way if requested, and at the end collect
 # them all into a future.
-async def read_decode_lines(reader: asyncio.StreamReader, q: 'asyncio.Queue[T.Optional[str]]',
-                            console_mode: ConsoleUser) -> str:
+async def read_decode_lines(reader: asyncio.StreamReader,
+                            q: 'asyncio.Queue[T.Optional[str]]') -> str:
     stdo_lines = []
     try:
         while not reader.at_eof():
             line = decode(await reader.readline())
             stdo_lines.append(line)
-            if console_mode is ConsoleUser.STDOUT:
-                print(line, end='', flush=True)
```

Comment:
This is needed to flush the output on every line with `--verbose` mode, why are you removing it? Please use commit messages fully, not just the subject.

Reply:
Ah, I see that this is `read_decode_lines()` so the subject is incorrect; and because it is used with `self.runobj.needs_parsing == True`, the `console_mode` cannot be `ConsoleUser.STDOUT`.

Reply:
Indeed. The commit message was wrong.